notebooks/case2_observations_analysis.md

```diff
@@ -465,22 +480,41 @@ n_plot_sex.head()
 ```{code-cell} ipython3
 :clear_cell: true
 
-pivoted = n_plot_sex.pivot_table(columns="sex", index="verbatimLocality", values="count")
+pivoted = n_plot_sex.pivot(columns="sex", index="verbatimLocality", values="count")
```

Comment:
Since we already did a groupby, all sex/locality combinations are unique, and this is actually a pivot and not pivot_table (the result is the same, since taking the "mean" of a single row is the same)